### DISCUSSION TITLE
ROI calculation criteria

### DISCUSSION THREAD

**Discussion by sonriks6** [google/meridian#294] [General]
Hey, had a question regarding the ROI calculation method, we input the 'revenue per kpi' as the product price and not the net profit, would make sense to inform about the margin instead? In our company we use to calculate ROI only on media driven sales by the net margin you get from each sales, then divide by the money invested in media. In this way the ROI used by Meridian is scaled by the ratio Margin/Revenue, right?

**Comment by brendalewisprice** [comment]
As far as I understand your approach, it does sound like you are fitting the same model, but just scaling the KPI by Margin / Revenue when compared to Meridian's approach.

Since your ROI definition is slightly different than the ROI definition used in Meridian, you will need to consider different custom ROI priors for your media that make sense. For example, in your case, ROI less than 1 but greater than 0 is still a profit, but the default ROI priors in Meridian assume that only ROI > 1 represents profit.

Adjusting the ROI priors to better reflect your prior understanding of ROI should be enough to get the model to work for you.

**Reply by sonriks6** [reply]
Thanks Brenda, yes this is the case; we've run a new model incorporating the revenue per sale, obviously the ROI is x times higher es we expected, so is time to explain this to stakeholders ;)

**Comment by sonriks6** [comment]
Closed as solved
